src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/Semantic/RazorSemanticTokensEndpoint.cs

```diff
@@ -94,19 +97,33 @@ public async Task<SemanticTokens> Handle(SemanticTokensRangeParams request, Canc
             return edits;
         }
 
-        public RegistrationExtensionResult GetRegistration()
+        public SemanticTokensRegistrationOptions GetRegistrationOptions()
         {
-            var semanticTokensOptions = new SemanticTokensOptions
+            return new SemanticTokensRegistrationOptions
```

Comment:
/cc @gundermanc since I wasn't aware of the spec changfe for semantic tokens but we'll now support old & new 😄

Reply:
Admittedly took me a bit by surprise as well. I have filed a bug to react. We'll plan on supporting both method names until everyone moves over.

Depending on my capacity, this may land in 16.8 Preview 3 or Preview 4. https://devdiv.visualstudio.com/DevDiv/_workitems/edit/1181012